sbnobj/Common/Reco/CRUMBSResult.h

```diff
@@ -12,14 +12,20 @@ namespace sbn {
   class CRUMBSResult {
   public:
 
-    CRUMBSResult(float score = default_float, float tpc_CRFracHitsInLongestTrack = default_float, float tpc_CRLongestTrackDeflection = default_float, 
+    CRUMBSResult(float score = default_float, float ccnumuscore = default_float, float ccnuescore = default_float, float ncscore = default_float, 
+		 float bestscore = default_float, int bestid = default_int, float tpc_CRFracHitsInLongestTrack = default_float, float tpc_CRLongestTrackDeflection = default_float, 
 		 float tpc_CRLongestTrackDirY = default_float, int tpc_CRNHitsMax = default_int, float tpc_NuEigenRatioInSphere = default_float, 
 		 int tpc_NuNFinalStatePfos = default_int, int tpc_NuNHitsTotal = default_int, int tpc_NuNSpacePointsInSphere = default_int, 
 		 float tpc_NuVertexY = default_float, float tpc_NuWeightedDirZ = default_float, float tpc_StoppingChi2CosmicRatio = default_float, 
 		 float pds_FMTotalScore = default_float, float pds_FMPE = default_float, float pds_FMTime = default_float, float crt_TrackScore = default_float, 
 		 float crt_HitScore = default_float, float crt_TrackTime = default_float, float crt_HitTime = default_float);
     
     float score;                         //!< CRUMBS result
```

Comment:
Same comment from the `sbnanaobj` review:
Ideally we would change `score` to something more descriptive now that there are other scores, but I guess we want to keep backwards compatibility so expanding the comment to be a bit more descriptive will have to suffice.